leafsmart-front-end/src/components/About.js

```diff
@@ -1,17 +1,84 @@
-import React from 'react'
+import React from "react";
+import { useNavigate } from "react-router-dom";
 import Nav from "./Nav";
 
+
 const About = () => {
+  const navigate = useNavigate();
+  
   return (
-    <main className="page-container">
+    <main className="pl-44 pr-32">
       <Nav />
-      <section className="bg-Isabelline">
-        <div>
-          <h1>About</h1>
+        <div className="bg-Isabelline py-5 h-screen">
+          <div className="mb-8 text-center">
+          <img
+              className="grid justify-items-start bg-White rounded-full h-auto w-32 mx-auto mt-1"
+              src="/LeafSmartDark.png"
+              alt="Lgo"
+            />
+            <h1 className="text-8xl">LeafSmart</h1>
+            <p className="text-normal italic mb-2">
+              The place to start thinking of travelling!
+            </p>
+            <p>
+              LeafSmart is a independantly built app focusing on the
```

Comment:
Just have a less wordy suggestion:
LeafSmart focuses on providing city data for our users' convenience. With technology at their fingertips, our users can find all they need to know about the locations they aim to travel to.

Reply:
also means don't need to /br potentially

Reply:
Updated this blurb per your request!